### PR TITLE
Fix refresh-styles script.

### PR DESCRIPTION
**Pull request by azizova-leyla**
### Description
1. Fixing the name of the docket container. 
2. Fixing the version of tailwindcss (not sure why, but it fails to install with current setup). 

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

**Comment by shanemc-goog** [comment]
Can you add more to the description, including the problem and how this addresses it?

**Review by goldblatt — DISMISSED** [review]

File: bin/refresh-styles [hunk 1]
```diff
@@ -8,7 +8,8 @@ docker run -it --rm \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/server:/usr/src/server \
   --entrypoint npx \
-  civiform \
+  civiform/civiform-dev:latest \
   tailwindcss build \
   -i ./app/assets/stylesheets/styles.css \
   -o ./public/stylesheets/tailwind.css
+
```
Comment (shanemc-goog) [inline bin/refresh-styles:15]: No newline at end of file

**Review by shanemc-goog — DISMISSED** [review]

**Review by kmcnellis — DISMISSED** [review]

**Review by bion — DISMISSED** [review]

File: bin/refresh-styles (hunk 1 above)
Comment (azizova-leyla) [reply bin/refresh-styles]: done

**Review by kmcnellis — APPROVED** [review]

**Review by bion — DISMISSED** [review]

File: bin/refresh-styles [hunk 2]
```diff
@@ -8,7 +8,7 @@ docker run -it --rm \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v $(pwd)/server:/usr/src/server \
   --entrypoint npx \
-  civiform \
+  civiform/civiform-dev:latest \
   tailwindcss build \
   -i ./app/assets/stylesheets/styles.css \
-  -o ./public/stylesheets/tailwind.css
+  -o ./public/stylesheets/tailwind.css
```
Comment (shanemc-goog) [inline bin/refresh-styles:14]: new line is still needed.

Comment (azizova-leyla) [reply bin/refresh-styles]: I think I did it right this time.

**Review by bion — APPROVED** [review]

**Review by bion — APPROVED** [review]

**Review by kmcnellis — APPROVED** [review]

**Pull request merged**